Admin/Pool.php

```diff
@@ -20,22 +20,49 @@
  */
 class Pool
 {
+    /**
+     * @var ContainerInterface|null
+     */
     protected $container = null;
 
+    /**
+     * @var array
+     */
     protected $adminServiceIds = array();
 
+    /**
+     * @var array
```

Comment:
@Soullivaneuh There is an usage: https://github.com/core23/SonataAdminBundle/blob/phpdoc/Admin/Pool.php#L85-L96 and it is used to generate the side menu